src/build.py

```diff
@@ -625,19 +626,36 @@ def Summary(repos):
     UploadToCloud('lkgr', 'git/lkgr', 'lkgr')
 
 
+def ParseArgs():
+  parser = argparse.ArgumentParser(
+      description='Wasm waterfall top-level CI script')
+  parser.add_argument('--no-sync', dest='sync',
+                      default=True, action='store_false',
+                      help='Skip fetching and checking out source repos')
+  parser.add_argument(
+      '--no-build', dest='build', default=True, action='store_false',
+      help='Skip building source repos (also skips V8 and LLVM unit tests)')
+  return parser.parse_args()
+
+
 def main():
+  options = ParseArgs()
   Clobber()
   Chdir(SCRIPT_DIR)
   Mkdir(WORK_DIR)
-  Remove(INSTALL_DIR)
-  repos = SyncRepos()
-  LLVM()
-  V8()
-  Sexpr()
-  OCaml()
-  Spec()
-  Binaryen()
-  ArchiveBinaries()
+  repos = {} if not options.sync else SyncRepos()
```

Comment:
I'd still generate content for `repos` by splitting out the `Sync` part from the one that goes through and figures out where each repo is. That gets printed out by `Summary` and makes it easier to figure out WTH was tested.